README.md

```diff
@@ -326,6 +326,26 @@ Obtain SMTP credentials from a provider for transactional emails. Set the SMTP_U
 
 <hr>
 
+<img src="https://upload.wikimedia.org/wikipedia/commons/4/44/Microsoft_logo.svg" height="50">
```

Comment:
Have you manually gone thru these to repeat them?  Usually folks write these kind of instructions after they have already finished coding and sometimes they don't match what a new user would go thru.